tools/code_format/check_format.py

```diff
@@ -1160,12 +1162,11 @@ def PooledCheckFormat(path_predicate):
     # to be rewritten by other multiprocessing pooled processes.
     PooledCheckFormat(lambda f: not isBuildFile(f))
     PooledCheckFormat(lambda f: isBuildFile(f))
-
     error_messages += sum((r.get() for r in results), [])
 
   if checkErrorMessages(error_messages):
-    print("ERROR: check format failed. run 'tools/code_format/check_format.py fix'")
-    sys.exit(1)
+      print("ERROR: check format failed. run 'tools/code_format/check_format.py fix'")
+      sys.exit(1)
```

Comment:
?

Reply:
really sorry for this. seems issue with my local env as it could not be catch.